src/app.js

```diff
@@ -1 +1,42 @@
 'use strict';
+
+const minimist = require('minimist');
+
+const inputArray = minimist(process.argv.slice(2))._;
```

Comment:
what if args provided are not numbers?
of not all are numbers?
let's filter out non-numerics
and could make sense to `.map(value => Number(value))`